src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/diagnostic_settings/subscription/_create.py

```diff
@@ -16,6 +16,9 @@
 )
 class Create(AAZCommand):
     """Create subscription diagnostic settings for the specified resource.
+
+    :example: Create diagnostic settings for a subscription with EventHub.
+        az monitor diagnostic-settings subscription create -n {name} --location westus --event-hub- auth-rule {eventHubRuleID} --storage-account {storageAccount} \ --logs '[ { "category": "Security", "enabled": true }, { "category": "Administrative", "enabled": true }, { "category": "ServiceHealth", "enabled": true }, { "category": "Alert", "enabled": true }, { "category": "Recommendation", "enabled": true }, { "category": "Policy", "enabled": true }, { "category": "Autoscale", "enabled": true }, { "category": "ResourceHealth", "enabled": true } ]'
```

Comment:
please use shorthand sytax in example
```suggestion
        az monitor diagnostic-settings subscription create -n {name} --location westus --event-hub- auth-rule {eventHubRuleID} --storage-account {storageAccount} \ --logs "[{category:Security,enabled:true},{category:Administrative,enabled:true},{category:ServiceHealth,enabled:true},{category:Alert,enabled:true},{category:Recommendation,enabled:true},{category:Policy,enabled:true},{category:Autoscale,enabled:true},{category:ResourceHealth,enabled:true}]"
```

Reply:
Please use fake name instead of `{xxxx}` in examples